### PR TITLE
Update defender-endpoint-plan-1-2.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated licensing information for P1 and P2

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8800

File: microsoft-365/security/defender-endpoint/defender-endpoint-plan-1-2.md [hunk 1]
```diff
@@ -44,6 +44,23 @@ The following table summarizes what's included in each Defender for Endpoint pla
 > [!IMPORTANT]
 > The standalone versions of Defender for Endpoint Plan 1 and Plan 2 do not include server licenses. To onboard servers, such as endpoints running Windows Server or Linux, you'll need Defender for Servers Plan 1 or Plan 2 as part of the [Defender for Cloud](/azure/defender-for-cloud/defender-for-cloud-introduction) offering. To learn more. see [Overview of Microsoft Defender for Servers](/azure/defender-for-cloud/defender-for-servers-introduction).
 
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/defender-endpoint-plan-1-2.md:47]:
```suggestion
```

File: microsoft-365/security/defender-endpoint/defender-endpoint-plan-1-2.md [hunk 2]
```diff
@@ -44,6 +44,23 @@ The following table summarizes what's included in each Defender for Endpoint pla
...
+### Microsoft Defender for Endpoint Plan 2 (P2)
+
+Microsoft Defender for Endpoint P2, which was previously called Microsoft Defender for Endpoint, is available as a standalone license and as part of the following plans:
+
+Windows 11 Enterprise E5/A5
+Windows 10 Enterprise E5/A5
+Microsoft 365 E5/A5/G5 (which includes Windows 10 or Windows 11 Enterprise E5)
+Microsoft 365 E5/A5/G5/F5 Security
+Microsoft 365 F5 Security & Compliance
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/defender-endpoint-plan-1-2.md]:
```suggestion
- Windows 11 Enterprise E5/A5
- Windows 10 Enterprise E5/A5
- Microsoft 365 E5/A5/G5 (which includes Windows 10 or Windows 11 Enterprise E5)
- Microsoft 365 E5/A5/G5/F5 Security
- Microsoft 365 F5 Security & Compliance
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**